src/css/custom.css

```diff
@@ -471,14 +490,46 @@ nav.menu {
   }
 
   .navbar {
-    height: 4.5rem;
+    height: var(--ifm-navbar-height-sm);
   }
 
   .navbar__logo {
-    height: 1.75rem;
+    height: var(--ifm-navbar-logo-height-sm);
   }
 
   .sidebar-search-container {
     display: none;
   }
+
+  .sidebar-search-container {
+    position: sticky;
+    top: 100px;
+    max-height: 100vh;
+    transition: all 250ms ease-in-out;
+    z-index: 99;
+  }
+
+  .navbarHidden_jGov {
+    transform: none;
+  }
+}
+
+@media (max-width: 600px) {
+  .algolia-autocomplete span[role="listbox"] {
+    display: block !important;
+    max-height: 90% !important;
+    max-width: 100% !important;
+    position: fixed !important;
+    top: 90px !important;
+    left: 0px !important;
+    width: 600px !important;
+    height: 100vh !important;
+    z-index: 100;
+    background: transparent !important;
+    overflow-y: auto !important;
```

Comment:
should not use `!important`

Reply:
using important only for left and top values as the aglolia search has added important for these styles values